admin-dev/themes/default/template/header.tpl

```diff
@@ -400,7 +400,9 @@
       </div>
 {/if}
 
+{if $display_header}
```

Comment:
If I'm right, you're already in the `if $display_header` which begins on line 110 :thinking: